sdks/python/apache_beam/testing/benchmarks/cloudml/requirements.txt

```diff
@@ -15,5 +15,6 @@
 #    limitations under the License.
 #
 
+dill
```

Comment:
Interesting - I see that this does indeed cause the correct test set to run, and avoids https://github.com/apache/beam/blob/b8f0d5fc12bd2585d8f3a434eab0026e54d73cf1/sdks/python/apache_beam/examples/ml_transform/ml_transform_it_test.py#L34

Do you know which import in that block requires dill?